colander_tools/timezone.py

```diff
@@ -0,0 +1,38 @@
+import pytz
+from pytz import exceptions
+
+import colander
+
+
+class TimezoneType(colander.SchemaType):
+    def serialize(self, node, appstruct):
+        if appstruct is colander.null:
+            return colander.null
+
+        try:
+            pytz.timezone(appstruct)
+        except exceptions.UnknownTimeZoneError:
+            raise colander.Invalid(
+                node,
+                colander._(
+                    '"${val}" is not a known timezone',
+                    mapping={"val": appstruct},
+                ),
+            )
+
+        return str(appstruct)
```

Comment:
Hm. That should be entitled to get a pytz tzinfo object as the `appstruct` no? If it is not a timezone object, just raise Invalid? If it is, return `appstruct.zone`.